pnpm-lock.yaml

```diff
@@ -9025,7 +9025,7 @@ packages:
     resolution: {integrity: sha512-rXckvqPBB0Krifk5rn/5LvQGmyXwCUpBfmTwbkQNBY9JY8RSl3b8OftBNEYxg4+SWUhEKcPifgope28uL9inlA==}
     engines: {node: '>=14.16'}
     peerDependencies:
-      '@types/react': 17.0.2
+      '@types/react': '>=18.0.0'
```

Comment:
Are these changes related? It's not the first time I see them, it could be changing depending on our local config/versions.

Reply:
I don't get any change now if I run `pnpm i` from main 🤷